### PR TITLE
Locales(uk_UA, ru_RU)

### PR DESCRIPTION
**Pull request by s-chizhik**
Hi. I'd like to add two locales.

### CODE REVIEW

File: src/Locales/ru_RU.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    "monthsShort"   => explode('_', 'Янв_Фев_Мар_Апр_Май_Июн_Июл_Авг_Сен_Окт_Ноя_Дек'),
+    "weekdays"      => explode('_', 'Понедельник_Вторник_Среда_Четверг_Пятница_Суббота_Воскресенье'),
+    "weekdaysShort" => explode('_', 'Пн_Вт_Ср_Чт_Пт_Сб_Вс'),
+    "calendar"      => array(
+        "sameDay"  => '[Сегодня]',
+        "nextDay"  => '[Завтра]',
+        "lastDay"  => '[Вчера]',
+        "lastWeek" => '[Прошлая неделя] l',
+        "sameElse" => 'l',
+        "withTime" => '[at] H:i',
```
Comment (fightbulc) [inline src/Locales/ru_RU.php]: What about the time?

File: src/Locales/uk_UA.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    "monthsShort"   => explode('_', 'Січ_Лют_Бер_Кві_Тра_Чер_Лип_Сер_Вер_Жов_Лис_Гру'),
+    "weekdays"      => explode('_', 'Понеділок_Вівторок_Середа_Четвер_П\'ятниця_Субота_Неділя'),
+    "weekdaysShort" => explode('_', 'Пн_Вт_Ср_Чт_Пт_Сб_Нд'),
+    "calendar"      => array(
+        "sameDay"  => '[Сьогодні]',
+        "nextDay"  => '[Завтра]',
+        "lastDay"  => '[Вчора]',
+        "lastWeek" => '[Минолого тижня] l',
+        "sameElse" => 'l',
+        "withTime" => '[at] H:i',
```
Comment (fightbulc) [inline src/Locales/uk_UA.php]: same as above. Time translation is missing.

**Comment by fightbulc** [comment]
Hey there. Love to see that the list of translations is growing. I had to remarks above. As soon as they are in I'll merge. Thanks!

**Comment by s-chizhik** [comment]
@fightbulc ok, i got it. I'll make it as soon as i can. Maybe you would provide unit test example to test common cases and prevent such mistakes? I saw `MomentGermanLocaleTest` but it does not cover all use-cases.

**Comment by fightbulc** [comment]
@s-chizhik Yes, I do have to take some time and implement some testing. Time is as always the challenge here.

**Comment by s-chizhik** [comment]
Hi. I noticed that there is no support of `accusative` and `nominative` versions of months, weeks, etc. So some calendar methods can give not fully correct result for this two locales. Is it ok? If you'll add this feature soon, I going to update locale. Or maybe I'll add feature and fix locales?

**Comment by fightbulc** [comment]
Hey there. Sure, man. I am happy if moment gets more sophisticated. Looking forward. Cheers

**Comment by fightbulc** [comment]
Hey there. Going to close this for now. Send a new request when you found the time. Thanks.

**Pull request closed without merging**